### PR TITLE
Add error check when creating binding

### PR DESCRIPTION
**Pull request by mkibbe1993**

### CODE REVIEW

**Comment by duglin** [comment]
LGTM if CI is happy

File: pkg/controller/catalog/server/controller.go [hunk 1]
```diff
@@ -142,6 +142,10 @@ func (c *controller) CreateServiceBinding(in *scmodel.ServiceBinding) (*scmodel.
 		Parameters: in.Parameters,
 	}
 	sbr, err := client.CreateServiceBinding(to.ID, in.ID, createReq)
+	if err != nil {
+		log.Printf("Failed to create service binding: %v\n", err)
```
Comment (arschles) [inline pkg/controller/catalog/server/controller.go]: use glog for this

Comment (MHBauer) [inline pkg/controller/catalog/server/controller.go]: A conversion from log to glog should all be done at one time. I'd prefer not to mix this error check with converting logging to glog. 

We have #168 to track.

**Comment by MHBauer** [comment]
LGTM as is.

**Comment by mkibbe1993** [comment]
Now that Brendan's [#191](https://github.com/kubernetes-incubator/service-catalog/pull/191) has updated the other logs in this file, I also updated this PR to glog as well.

File: pkg/controller/catalog/server/controller.go (hunk 1 above)
Comment (arschles) [inline pkg/controller/catalog/server/controller.go]: fine with me

**Comment by arschles** [comment]
LGTM3, merging

**Pull request merged**